bench.sh

```diff
@@ -27,71 +21,69 @@ while [[ ${1} ]]; do
     fi
 done
 
-if [[ -z ${DRIVER} ]]; then
-    echo ${USAGE} >&2
+if [ -z "${DRIVER}" ]; then
+    echo "${USAGE}" >&2
     return 1
 fi
 
-CPU=$[`grep -c ^processor /proc/cpuinfo`/2]
-#MEM=$[`grep -E 'MemAvailable' /proc/meminfo | rev | cut -d " " -f 2 | rev`/1024/2-500]
-MEM=$[`grep -E 'MemAvailable' /proc/meminfo | rev | cut -d " " -f 2 | rev`/1024-800]
-PHP_CPU=
-DB_CPU=
-
-COUNTER=0
-while [  $COUNTER -lt ${CPU} ]; do
-    if [ $COUNTER -ne 0 ]; then
-        PHP_CPU=${PHP_CPU},
-        DB_CPU=${DB_CPU},
-    fi
-
-    PHP_CPU=${PHP_CPU}$[COUNTER*2]
-    DB_CPU=${DB_CPU}$[COUNTER*2+1]
-    let COUNTER=COUNTER+1
-done
+export DRIVER=${DRIVER}
```

Comment:
@prolic I've added an export so the driver env variable should be now availabe in PHP. Can you test it please? If this not works, can you try to use `php -dDRIVER=${DRIVER} src`?

Reply:
ok, but what about the time measurement problem? All drivers have about same performance testing with your benchmark modifications.